main.nf

```diff
@@ -3,7 +3,7 @@ nextflow.enable.dsl = 2
 
 //runs test config for Agora
 process AGORA_DATA_RUN {
-
+    debug = true
```

Comment:
Do you want to keep this on `debug = true`?

Reply:
I do not. Thanks!